app/collections/create-form.jsx

```diff
@@ -77,6 +89,18 @@ class CollectionsCreateForm extends React.Component {
             placeholder="Collection Name"
           />
         </label>
+        <label>
+          <textarea
+            className="collection-name-input"
```

Comment:
RFC: I'm currently using the `.collection-name-input` class for both the name input and the description input styles - which seems not right. Would it be better to change the name to something like `.collection-inputs`? or make a separate css class for `.collection-description-input`? 

Right now, styling the name and description inputs with the same selector seems to be working just fine, but I'm not sure what's best practice for this.

Reply:
This is something that the BEM can help us with. Let's do a few things:

- Rename this file to be consistent with the name of the component
- The top level class name `collections-create-form` is the block
- Inputs are elements in the block, so following BEM convention: `collections-create-form__input`
- Any variants should use the modifier convention: `collections-create-form__input--description`